tekton/pipeline_run.go

```diff
@@ -48,24 +49,56 @@ var (
 	ReleaseWorkspaceLabel = fmt.Sprintf("%s/%s", releaseLabelPrefix, "workspace")
 )
 
-// CreateReleasePipelineRun creates a PipelineRun from a given Release and ReleaseStrategy.
-// Release details are added to the labels of the new PipelineRun to be able to reference it later on.
-func CreateReleasePipelineRun(release *v1alpha1.Release, releaseStrategy *v1alpha1.ReleaseStrategy) *tektonv1beta1.PipelineRun {
-	return &tektonv1beta1.PipelineRun{
+type ReleasePipelineRun struct {
```

Comment:
It is not clear to me why we are defining this struct instead of just using the normal `tektonv1beta1.PipelineRun`

Reply:
You cannot add new methods in a different file from where the receiver is defined.

Reply:
That's unfortunate. Can you comment pretty much exactly that before defining this then?

Reply:
Can do.